doc/tutorials/plutus-playground.rst

```diff
@@ -68,98 +78,106 @@ The blockchain section shows all the transactions that were produced during the
 Each simulation is run on a fresh blockchain.
 The "Hello, World" simulation only results in a single transaction.
 This is the initial transaction that distributes some funds to the simulated agents.
-We can click the box that says "Slot 0, Tx 0" to examine the transaction.
+You can click the box that says "Slot 0, Tx 0" to examine the transaction.
```

Comment:
I just merged a PR that makes the first transaction preselected by default.

Reply:
Good catch!